src/components/ProfilePage.tsx

```diff
@@ -177,7 +212,9 @@ export const ProfilePage = (props: ProfilePageProps) => {
                         <div className='grid-container'>
                             <div className='containers'>
                                 {!editing ? 
-                                    <Typography variant='headline3'>{`${userBundle?.userProfile?.firstName} ${userBundle?.userProfile?.lastName}`}</Typography>
+                                    <Typography variant='headline3'>
+                                        {`${userBundle?.userProfile?.firstName} ${userBundle?.userProfile?.lastName}`}
+                                    </Typography>
                                 : 
                                 <FormGroup style={{display:'inline-block'}}>
                                     {EditField('First name', firstName, setFirstName)}
```

Comment:
Since EditField returns JSX, it should follow React conventions, so its arguments should be passed as props and it should itself be rendered using JSX instead of calling the function.